ivy/functional/frontends/jax/numpy/fft.py

```diff
@@ -48,3 +48,21 @@ def ifft2(a, s=None, axes=(-2, -1), norm=None):
     if norm is None:
         norm = "backward"
     return ivy.array(ivy.ifft2(a, s=s, dim=axes, norm=norm), dtype=ivy.dtype(a))
+
+
+@with_unsupported_dtypes({"2.5.2 and below": ("float16", "bfloat16")}, "paddle")
+@to_ivy_arrays_and_back
+def rfft(a, n=None, axis=-1, norm=None):
+    if norm is None:
+        norm = "backward"
+    if n is None:
+        n = len(a)
+    if ivy.current_backend_str() == "tensorflow":
```

Comment:
Hi @AbdullahSabry,
it's not the ideal way to deal with tensorflow backend. I think either we've to use `with_unsupported_dtypes` for tensorflow or restricted the data types in the tensorflow backend for fft function.

Reply:
Pushed changes to make the casting happen on all backends to avoid confusion